README.md

```diff
@@ -0,0 +1,120 @@
+| `dovecot_dsync_max_connections`   | `10`                    | Maximum parallel dsync connections to spawn                                              |
+| `dovecot_dsync_port`              | `12345`                 | Dsync port to use                                                                        |
+| `dovecot_dsync_password`          | (:heavy_check_mark:)    | (Only if using dsync) Shared secret to establish dsync connection                        |
+| `dovecot_dsync_parameters`        | `-d -N -l 3000 -U`      | Parameters to set for dsync child processes                                              |
+| `dovecot_dsync_sync_timeout`      | `null`                  | Set to a non-null value to enable synchronous replication                                |
+| `dovecot_dsync_target`            | `null`                  | Dsync target to replicate with                                                           |
+| `dovecot_imap_hibernate_timeout`  | `5s`                    | Time for clients to idle before being pushed to the hibernation process                  |
+| `dovecot_imap_min_procs`          | `4`                     | Minimum amount of IMAP processes to keep available                                       |
+| `dovecot_private_prefix`          | `null`                  | Prefix for the private dovecot namespace                                                 |
+| `dovecot_private_mailboxes`       | `{}`                    | Mailbox configurations for the private mailboxes (see below)                             |
+| `dovecot_rspamd_connect`          | `null`                  | Rspamd host to connect to for reporting spam                                             |
+| `dovecot_rspamd_password`         | `null`                  | Rspamd password to use when reporting spam                                               |
+| `dovecot_fts_autoindex`           | `true`                  | Whether to enable autoindexing for new mails                                             |
+| `dovecot_protected_mailboxes`     | `[]`                    | List of folder names that should be non-deletable and impossible to create subfolders of |
+| `dovecot_extra_acls`              |                         | Extra text to append to the ACLs file                                                    |
+
+### mailbox configuration
+| Name          | Default / Mandatory | Description                                             |
+|:--------------|:-------------------:|:--------------------------------------------------------|
+| `special_use` |                     | IMAP special-use flag                                   |
+| `auto`        |                     | `auto` value for Dovecot                                |
+| `autoexpunge` |                     | Autoexpunge value (is applied to the IMAP service only) |
```

Comment:
You could mention that the value should be a timeframe

Reply:
and what is the measurment unit